catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/Metacard.java

```diff
@@ -93,6 +93,9 @@
   /** {@link Attribute} name for accessing the title of the {@link Metacard}. <br> */
   String TITLE = "title";
 
+  /** {@link Attribute} name for accessing the owner of the {@link Metacard}. <br> */
+  String OWNER = "owner";
```

Comment:
❓ Should we be touching Metacard.java? I thought we were moving over to the newer Attributes (eg, CoreAttributes)
❓ Do we not have a string for owner in the SecurityAttributes?